python/src/energuide/interpreter.py

```diff
@@ -113,3 +114,16 @@ def from_row(cls, row: InputData) -> 'ParsedDwellingDataRow':
             postal_code=row['CLIENTPCODE'],
             forward_sortation_area=row['CLIENTPCODE'][0:3]
         )
+
+
+def read(filename: str) -> typing.Iterator[InputData]:
+    with open(filename, 'r') as csvfile:
+        reader = csv.reader(csvfile)
+        #pylint: disable=stop-iteration-return
+        columns = next(reader)
+
+        def to_dict(row) -> InputData:
```

Comment:
You could just embed this dict comprehension directly in the `for` loop before and get rid of the pylint exception. Or, define the inner `to_dict` function before the `open` command` - that might also get rid of the linter error.